index.d.ts

```diff
@@ -256,6 +256,7 @@ declare module "leap-core" {
     getMerkleTree(): MerkleTree;
     merkleRoot(): string;
     proof(tx: Tx<any>): Proof;
+    static periodBlockRange(blockNumber: number): Array<number>[2];
```

Comment:
Why not tupple (`[number, number]`)?

Reply:
blindly copied the line from the old typdef for `helpers`. Thanks for the hint